pkg/arguments/arguments.go

```diff
@@ -216,3 +230,183 @@ func ApplyPathArg(request *sdk.Request, value string) error {
 	}
 	return nil
 }
+
+// ifInteractive is a helper running the given function if --interactive is set.
+func ifInteractive(fs *pflag.FlagSet, then func() error) error {
+	interactive, err := fs.GetBool("interactive")
+	if err != nil {
+		return fmt.Errorf(`BUG: no such flag "interactive"`)
+	}
+	if !interactive {
+		return nil
+	}
+	return then()
+}
+
+// PromptBool sets a bool flag value interactively, unless already set.
+func PromptBool(fs *pflag.FlagSet, flagName string) error {
+	return ifInteractive(fs, func() error {
+		value, err := fs.GetBool(flagName)
+		if err != nil {
+			return fmt.Errorf("BUG: %v", err)
```

Comment:
I think just printing the error is enough without the "BUG:"

Reply:
Will do.  BTW what do you think of `panic()` in these cases, in similar spirit to #187?  can only happen if caller mis-spelled flag name or mistaked flag's type.